src/System.CommandLine.Tests/VersionOptionTests.cs

```diff
@@ -155,33 +155,17 @@ public void Version_option_is_not_added_to_subcommands()
                   .BeEmpty();
         }
 
-        [Fact]
-        public async Task Version_not_added_if_it_exists()
-        {
-            // The RootCommand ctors adds VersionOption by default.
-            // Users might not know if or just want to customize it.
-            RootCommand commandWithDuplicates = new()
-            {
-                new VersionOption(),
-                new VersionOption(),
-            };
-            CommandLineConfiguration configuration = new (commandWithDuplicates)
-            {
-                Output = new StringWriter()
-            };
-
-            await configuration.InvokeAsync("--version");
-
-            configuration.Output.ToString().Should().Be($"{version}{NewLine}");
-        }
-
         [Fact]
         public async Task Version_can_specify_additional_alias()
         {
-            RootCommand rootCommand = new()
+            RootCommand rootCommand = new();
+
+            for (int i = 0; i < rootCommand.Options.Count; i++)
             {
-                new VersionOption("-v", "-version"),
-            };
+                if (rootCommand.Options[i] is VersionOption)
+                    rootCommand.Options[i] = new VersionOption("-v", "-version");
```

Comment:
This is a general comment and doesn't need to block merging.

The API through beta4 treated the arguments to `Option<T>` such that  `new VersionOption("-v", "-version")` would indicate `-version` is the description.

The beta5 API understands this as setting `Name` to `-v` and using `-version` as an alias. In general usage though, `-version` would be the name (being longer) and `-v` would be the alias.

Two observations:

* I've found this to be a gotcha in upgrading my code.
* This usage in tests, while perfectly valid, might mislead about the way in which the API is typically used.